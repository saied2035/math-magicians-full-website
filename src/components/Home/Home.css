@import url('https://fonts.cdnfonts.com/css/sailor-vito');

.home-text {
  color: #1b2021;
  font-weight: 600;
  font-family: 'Sailor Vito', sans-serif;
  letter-spacing: 1.5px;
}
