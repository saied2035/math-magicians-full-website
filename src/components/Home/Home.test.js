import React from 'react';
import renderer from 'react-test-renderer';

import Home from './Home';

it('Rendere to check navigation list', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
