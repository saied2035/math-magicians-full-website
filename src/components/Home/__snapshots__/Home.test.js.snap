// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendere to check navigation list 1`] = `
Array [
  <h2
    className="home-text pl4"
  >
    Welcome to our page
  </h2>,
  <p
    className="home-text pl4 f4 tl"
  >
    Mathematics is the cradle of all creations, without which the world cannot move an inch. Be it a cook or a farmer, a carpenter or a mechanic, a shopkeeper or a doctor, an engineer or a scientist, a musician or a magician, everyone needs mathematics in their day-to-day life. Even insects use mathematics in their everyday life for existence.
  </p>,
  <p
    className="home-text pl4 pr2 f4 tl"
  >
    Experience says learning mathematics can be made easier and enjoyable if our curriculum includes mathematical activities and games. Maths puzzles and riddles encourage and attract an alert and open-minded attitude among youngsters and help them develop clarity in their thinking. Emphasis should be laid on development of clear concept in mathematics in a child, right from the primary classes.
  </p>,
]
`;
